devTools/svgTester/export-graphs.ts

```diff
@@ -28,14 +37,37 @@ async function main(parsedArgs: parseArgs.ParsedArgs) {
         const directories = []
         for await (const entry of dir) {
             if (entry.isDirectory()) {
-                if (!targetConfigs.length) {
+                if (!targetConfigs.length && !targetChartTypes.length) {
+                    directories.push(entry.name)
+                    continue
+                }
+
+                if (targetConfigs.includes(entry.name)) {
                     directories.push(entry.name)
-                } else if (targetConfigs.includes(entry.name)) {
+                    continue
+                }
+
+                const configPath = path.join(inDir, entry.name, "config.json")
+                const config = JSON.parse(fs.readFileSync(configPath, "utf8"))
```

Comment:
```suggestion
                const config = await fs.readJson(configPath)
```
(this works because we're using [fs-extra](https://yarnpkg.com/package/fs-extra))